packages/koa-metrics/src/test/index.test.ts

```diff
@@ -0,0 +1,5 @@
+describe('koa-metrics', () => {
+  it('works', () => {
```

Comment:
I don't like leaving this empty D:

I'd really like to see us at least leave empty its defining the tests we intend to write if we're going to do them in a subsequent PR.